recipes/cime-env/meta.yaml

```diff
@@ -30,12 +30,12 @@ requirements:
     - pandas
     - matplotlib
     - jinja2
-    - json_tricks 3.11.0
+    - json_tricks 3.15.5
```

Comment:
@jhkennedy, do you recall if there was a reason to pin to a specific version of json_tricks?

Reply:
It looks like this is pinned in livvkit: https://github.com/LIVVkit/LIVVkit/issues/26

I don't see any evidence that json_tricks is used in CIME or E3SM, so I don't think it should be part of the metapackage.

Reply:
yeah, that's def. just a LIVVkit dep.